src/ResourceManager/Network/Commands.Network/ChangeLog.md

```diff
@@ -23,6 +23,11 @@
 
 ## Version 6.7.0
 * Updated cmdlet New-AzureRmVirtualNetworkGatewayConnection with support for switch ExpressRouteGatewayBypass
+* Added NetworkProfile functionality. New cmdlets added
```

Comment:
* Move this under `Current Version`
* There are more cmdlets added than listed here